### ISSUE TITLE
Career is still under development?

### ISSUE THREAD

**Issue by best-lucky1030** [CodeCrowCorp/mage-website#379]
Hi Contributors.
Looking around the website, `career` section is still under development?
This is just for only front-end? 
If then, what kind of framework you are using as back-end to save all info?

**Comment by gagan-suie** [comment]
yes we are still just in pre-beta phase building. this repo is only for frontend. backend is handled in another repo using cloudflare workers.

**Comment by best-lucky1030** [comment]
@gagan-suie 
thx for your reply.
If then, it's possible to collaborate on this project?

**Comment by gagan-suie** [comment]
Please apply to the job on upwork.

**Comment by best-lucky1030** [comment]
> Reference

Yup. @gagan-suie  
just applied via upwork. name is Alex

**Comment by best-lucky1030** [comment]
@gagan-suie 
any update?
